_test/test_gen_sqw_for_powders/test_combine_pow.m

```diff
@@ -95,6 +96,70 @@
             da(w2_1);
             close all
             %--------------------------------------------------------------------------------------------------
+        end
+        function obj = test_spher_cut_fine_grid(obj)
+            % Create sqw files, combine and check results
+            % -------------------------------------------
+            sqw_file_tot=fullfile(tmp_dir,'test_spher_cut_fine_grid.sqw');
+            cleanup_obj = [];
+            if ~isfile(sqw_file_tot)
```

Comment:
its extremely inconvenient to clear each time you run the test multiple times when debug the operations because 90% time is spent on preparation. And on Jenkins or if you do not care about the test it behaves fine